util.js

```diff
@@ -63,8 +63,9 @@ function showHelpForCommand (commandName) {
 
   try {
     console.log(fs.readFileSync(helpFilePath, 'utf8'))
+    exit()
```

Comment:
And there

Reply:
I think changing this line to be `exit(fs.readFileSync(helpFilePath, 'utf8'))` would cause help command to return a non-zero exit code. Is this OK?

I might be misinterpreting your change request, so please do let me know if so is the case.

Reply:
@francisbrito I am sorry for my mistake, I wanna add comment on [Line67](https://github.com/fastify/fastify-cli/pull/193/files#diff-63f3aa32c5cc5d42a250b081693e9dc2R67). I think is unnecessary to tell users two messages about the error